operate/client/src/modules/stores/autoMapping.test.ts

```diff
@@ -1,88 +0,0 @@
-/*
```

Comment:
Renamed to processInstanceMigrationMapping.test.ts

To see the diff, please check [this commit](https://github.com/camunda/camunda/pull/22179/commits/eb05ea3c92d6828bdbf9c6f5af3bdda05b6cd667)